rllib/env/multi_agent_env_runner.py

```diff
@@ -677,6 +652,56 @@ def stop(self):
         # Note, `MultiAgentEnv` inherits `close()`-method from `gym.Env`.
         self.env.close()
 
+    def _make_env(self):
+        """Creates a MultiAgentEnv (is-a gymnasium env)."""
+        env_ctx = self.config.env_config
+        if not isinstance(env_ctx, EnvContext):
+            env_ctx = EnvContext(
```

Comment:
Do we still need the `EnvContext` to start the gymnasium environments? `gymnasium` uses the `kwargs`

Reply:
Gym Envs don't need it, but RLlib users are so used to having these properties in the env config dict that it would break things for a lot of them:
```
config.worker_index = ...
config.num_workers = ...
config["some_setting_for_env_ctor"] = ...
```

Reply:
I see. It is sometimes a bit unintuitive to use a dicitonary instead of directly inserting variables like in `gymnasium`  directly.

Reply:
Yeah, I agree. It's historical and I don't want to break this API right now. We should discuss this in more detail. For now I just wanted to make the new stack behave just like the old in this regard.

Reply:
Understandable.